java/test/src/main/java/org/ray/api/test/ActorTest.java

```diff
@@ -1,152 +1,59 @@
 package org.ray.api.test;
 
-import java.util.ArrayList;
-import java.util.List;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.ray.api.Ray;
 import org.ray.api.RayActor;
 import org.ray.api.RayObject;
-import org.ray.api.RayRemote;
-import org.ray.api.UniqueID;
+import org.ray.api.annotation.RayRemote;
+import org.ray.api.function.RayFunc2;
+import org.ray.api.id.UniqueId;
 
 @RunWith(MyRunner.class)
 public class ActorTest {
 
   @RayRemote
-  public static Integer sayWorld(Integer n, RayActor<ActorTest.Adder> adder) {
-    RayObject<Integer> result = Ray.call(ActorTest.Adder::add, adder, 1);
-    return result.get() + n;
-  }
-
-  @Test
-  public void test() {
-
-    RayActor<ActorTest.Adder> adder = Ray.create(ActorTest.Adder.class);
-    Ray.call(Adder::set, adder, 10);
-    RayObject<Integer> result = Ray.call(Adder::add, adder, 1);
-    Assert.assertEquals(11, (int) result.get());
-
-    RayActor<Adder> secondAdder = Ray.create(Adder.class);
-    RayObject<Integer> result2 = Ray.call(Adder::add, secondAdder, 1);
-    Assert.assertEquals(1, (int) result2.get());
-
-    RayObject<Integer> result3 = Ray.call(Adder::add2, 1);
-    Assert.assertEquals(2, (int) result3.get());
+  public static class Counter {
 
-    RayObject<Integer> result4 = Ray.call(ActorTest::sayWorld, 2, adder);
-    Assert.assertEquals(14, (int) result4.get());
+    private int value = 0;
 
-    RayActor<Adder2> adder2 = Ray.create(Adder2.class);
-    Ray.call(Adder2::setAdder, adder2, adder);
-    RayObject<Integer> result5 = Ray.call(Adder2::increase, adder2);
-    Assert.assertEquals(1, (int) result5.get());
-
-    List list = new ArrayList<>();
-    list.add(adder);
-    Ray.call(Adder2::setAdderList, adder2, list);
-
-    RayObject<Integer> result7 = Ray.call(Adder2::testActorList, adder2);
-    Assert.assertEquals(14, (int) result7.get());
-
-    List tempList = new ArrayList<>();
-    tempList.add(result);
-    Ray.call(Adder::setObjectList, adder, tempList);
-    RayObject<Integer> result8 = Ray.call(Adder::testObjectList, adder);
-    Assert.assertEquals(11, (int) result8.get());
+    public int incr(int delta) {
+      value += delta;
```

Comment:
I think `increase` would be better.
But this is a test file, it's ok.

Reply:
`incr` is a commonly used in code as an abbreviation of `increase`. I think it would not cause ambiguity here.